configure.ac

```diff
@@ -42,7 +42,14 @@ GLIB_GSETTINGS
 
 #############################################################
 
-JH_CHECK_XML_CATALOG([http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl], [DocBook XSL Stylesheets])
+AC_ARG_ENABLE([docs],
+    [AS_HELP_STRING([--enable-docs], [Enable building documentation])],
+    [ENABLE_DOCS="${enableval}"], [ENABLE_DOCS='yes'])
+AM_CONDITIONAL(ENABLE_DOCS, [test x$ENABLE_DOCS = xyes])
+if test "x$ENABLE_DOCS" = "xyes"; then
+    JH_CHECK_XML_CATALOG([http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl], [DocBook XSL Stylesheets])
+fi
```

Comment:
If we were to try and be pedantic, this should be written as:

```
AS_IF([test "x$ENABLE_DOCS" = "xyes"],
      [JH_CHECK_XML_CATALOG([http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl],
                            [DocBook XSL Stylesheets])])
```

Reply:
Or use AM_COND_IF([ENABLE_DOCS], [... ])

Reply:
> AM_COND_IF()

Oh, I did not know about that one, thanks! :-)